bulk_webincli.py

```diff
@@ -1,57 +1,139 @@
+    )
+    parser.add_argument(
+        "-w",
+        "--webinCliPath",
+        help='Full path to Webin-CLI jar file. Default: "/webin-cli.jar" (for Docker)',
+        default="/webin-cli.jar",
+        type=str,
+    )
+    parser.add_argument(
+        "-u",
+        "--username",
+        help="Webin submission account username (e.g. Webin-XXXXX)",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-p",
+        "--password",
+        help="password for Webin submission account",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-g",
+        "--geneticContext",
+        help="Context for submission, options: genome, transcriptome, sequence, reads, taxrefset",
+        choices=["genome", "transcriptome", "sequence", "reads", "taxrefset"],
+        nargs="?",
+        required=True,
+    )
+    parser.add_argument(
+        "-s",
+        "--spreadsheet",
+        help="name of spreadsheet with metadata",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-d",
+        "--directory",
+        help="parent directory of data files",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-c",
+        "--centerName",
+        help="FOR BROKER ACCOUNTS ONLY - provide center name",
+        type=str,
+        required=False,
+    )
+    parser.add_argument(
+        "-m",
+        "--mode",
+        type=str,
+        help="options for mode are validate/submit",
+        choices=["validate", "submit"],
+        nargs="?",
+        required=False,
+    )
+    parser.add_argument(
+        "-pc",
+        "--parallel",
+        help="Run submissions in parallel and specify the number of cores/threads to use, maximum cores/threads=10",
+        type=int,
+        required=False,
+    )
+    parser.add_argument(
+        "-t",
+        "--test",
+        help="specify usage of test submission services",
+        action="store_true",
+    )
+    parser.add_argument(
+        "-a",
+        "--ascp",
+        help="Use Aspera (ascp needs to be in path) instead of FTP when uploading files.",
+        action="store_true",
+    )
 
     args = parser.parse_args()
 
     if args.mode is None:
-        args.mode = "validate"      # If no mode is provided, default to Webin-CLI validate mode
-    if args.directory is None:
-        args.directory=""
+        args.mode = (
+            "validate"  # If no mode is provided, default to Webin-CLI validate mode
+        )
     if args.centerName is None:
-        args.centerName=""
+        args.centerName = ""
     if args.parallel is None:
         args.parallel = False
     elif not 0 < args.parallel <= 10:
-        print('> ERROR: Invalid number of cores/threads provided. This value should be between 1 and 10 (inclusive).')
+        print(
+            "> ERROR: Invalid number of cores/threads provided. This value should be between 1 and 10 (inclusive)."
+        )
```

Comment:
I'd be more in favour of keeping this more compact and not spreading out the print statement onto three lines.